client/http/interface.go

```diff
@@ -768,12 +770,25 @@ func (c *client) GetSchedulers(ctx context.Context) ([]string, error) {
 	return schedulers, nil
 }
 
+// GetSchedulerConfig returns the configuration of the specified scheduler for pd cluster
+func (c *client) GetSchedulerConfig(ctx context.Context, name string) (map[string]any, error) {
+	var config map[string]any
+	uri := path.Join(SchedulerConfig, name, "list")
+	err := c.request(ctx, newRequestInfo().
+		WithName(getSchedulerConfig).
+		WithURI(uri).
+		WithMethod(http.MethodGet).
+		WithResp(&config))
+	if err != nil {
+		return nil, err
+	}
+	return config, nil
+}
+
 // CreateScheduler creates a scheduler to PD cluster.
-func (c *client) CreateScheduler(ctx context.Context, name string, storeID uint64) error {
-	inputJSON, err := json.Marshal(map[string]any{
-		"name":     name,
-		"store_id": storeID,
-	})
+func (c *client) CreateScheduler(ctx context.Context, name string, input map[string]any) error {
```

Comment:
We could consider keeping both API methods. Users who want to create a scheduler with customized data can use `CreateSchedulerWithInput`, while those who only need to provide a `name` and `storeID` can use `CreateScheduler`.